src/components/CurrencyLogo/index.tsx

```diff
@@ -33,16 +34,16 @@ function CurrencyLogo({
   style?: React.CSSProperties
 }) {
   const logoURI = currency instanceof WrappedTokenInfo ? currency?.logoURI : undefined
-  const uriLocations = useHttpLocations(logoURI)
+  const uriLocations = useHttpLocations(getProxyTokenLogo(logoURI))
```

Comment:
Bro need to serious re-design on this. Proxy logo only whitelist a small set of hosts and maintain by add by manually then release with complicated steps & approve. If you want to use it on all logos, most of them wont work. Please take carefully on this.

Reply:
@nguyenhoaidanh please help

Reply:
Fixed